src/current/v23.2/physical-cluster-replication-overview.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Physical Cluster Replication
+summary: Follow a tutorial to set up physical replication between a primary and standby cluster.
+toc: true
+docs_area: manage
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} CockroachDB physical cluster replication continuously sends all data at the byte level from a _primary_ cluster to an independent _standby_ cluster. Existing data and ongoing changes on the active primary cluster, which is serving application data, replicate asynchronously to the passive standby cluster.
```

Comment:
Question: Is it possible to configure multiple standby clusters with a single primary cluster?

Reply:
It is, though checking with eng, it doesn't seem like something we want to recommend, or suggest.